pkgs/development/libraries/podofo/0.10.x.nix

```diff
@@ -53,8 +53,12 @@ stdenv.mkDerivation (finalAttrs: {
     "-DCMAKE_BUILD_WITH_INSTALL_NAME_DIR=ON"
   ];
 
-  postInstall = ''
-    moveToOutput lib "$lib"
+  postPatch = ''
+    # Use GNU directories to fix multiple outputs
+    sed -i src/podofo/CMakeLists.txt \
+        -e 's/LIBDIRNAME/CMAKE_INSTALL_LIBDIR/g'
```

Comment:
At least this line can be swapped to substituteInPlace --replace-fail so that it doesn't silently break in the next update.

Reply:
Good call: it wasn't even necessary. The new versions are already using the standard GNU directories.